cmd/ocagent/main.go

```diff
@@ -152,13 +153,14 @@ func runOCInterceptor(addr string, sr spanreceiver.SpanReceiver) (doneFn func()
 	agenttracepb.RegisterTraceServiceServer(srv, oci)
 	go func() {
 		log.Printf("Running OpenCensus interceptor as a gRPC service at %q", addr)
-
-		// Not using log.Fatalf(srv.Serve(ln)) because CTRL-C will close the listener as the user requested
-		// and that log.Fatalf will produce an unrelated but misleading error:
-		//     "Failed to run OpenCensus interceptor: accept tcp 127.0.0.1:55678: use of closed network connection"
-		_ = srv.Serve(ln)
+		if err := srv.Serve(ln); err != nil {
```

Comment:
@pjanotti please take a look at that comment that you've deleted on why it was `_ = srv.Serve(ln)`

Reply:
The thing was that it was closing the listener from which the server depended upon, the preferred way is to let the server itself close the listener. This way there is no error message from the gRPC server.